api_products.md

```diff
@@ -11,25 +11,214 @@ information about api access.
+mechanism can also be used to model bundles like a bag of bread rolls
+or instore EANs simply naming the price of the product.
+
+The price of a weighable product depends on information encoded into a
+scannable code. There are three cases:
+
+- The code contains the measurement in the encoding unit. This
+  measurement gets converted into the reference unit which allows the
+  calculation of the price
+
+- The code contains the units in a prepackaged container or bag
+
+- The code contains the price
+
+This information is extracted using the [Code
+Templates](#code-template) described below.
+
+#### Supported Units
+
+The supported units are:
+
+| Abbreviation | Description                                                                                                    |
+|--------------|----------------------------------------------------------------------------------------------------------------|
+| ml           | Mililiter                                                                                                      |
+| dl           | Deciliter                                                                                                      |
+| l            | Liter                                                                                                          |
+| m3           | Cubic meter                                                                                                    |
+| cm3          | Cubic centimeter                                                                                               |
+| m2           | Square meter                                                                                                   |
+| cm2          | Square centimeter                                                                                              |
+| mm           | Millimeter                                                                                                     |
+| cm           | Centimeter                                                                                                     |
+| m            | Meter                                                                                                          |
+| kg           | Kilogram                                                                                                       |
+| g            | Gram                                                                                                           |
+| t            | Tonne                                                                                                          |
+| piece        | The encoded number should be interpreted as number of contained units, i.e. number of bread rolls in a package |
+| price        | The encoded number should be interpreted as total price, i.e. price of a package of different sausages         |
+
+
+### Code Template
+
+Code templates are a way to extract information embeded into a scanned
```

Comment:
Eher `out of` oder? Wir extrahieren ja Daten aus dem Code und packen sie nicht rein.